pkg/queue/workers/task_worker_test.go

```diff
@@ -93,6 +93,46 @@ func TestTaskWorkerWork(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
 	defer cancel()
 
+	t.Run("worker handles multiple tasks without stopping", func(t *testing.T) {
```

Comment:
Is this the right name for the test?

Reply:
I thought so, the purpose is to show that the worker doesn't stop after a successful task, similar to the worker not stopping after a failed test